pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -2,207 +2,643 @@
 
-```shell
-BOT_TOKEN=YourDiscordBotTokenHere
-BOT_API_KEY=badbot13m0n8f570f942013fc818f234916ca531
-REDDIT_CLIENT_ID=YourRedditClientIDHere
-REDDIT_SECRET=YourRedditSecretHere
-```
+We understand this is tedious and are working on a better solution for setting up test servers.
 
----
-# Run the project
+<div class="card">
+    <button type="button" class="card-header collapsible">
+        <span class="card-header-title subtitle is-6 my-2 ml-2">Why do you need a separate config file?</span>
+        <span class="card-header-icon">
+            <i class="fas fa-angle-down title is-5" aria-hidden="true"></i>
+        </span>
+    </button>
+    <div class="collapsible-content">
+        <div class="card-content">
+            While it's technically possible to edit <code>config-default.yml</code> to match your server, it is heavily discouraged.
+            This file's purpose is to provide the configurations the Python bot needs to run in the Python server in production, and should remain as such.
+            In contrast, the <code>config.yml</code> file can remain in your local copy of the code, and will be ignored by commits via the project's <code>.gitignore</code>.
+        </div>
+    </div>
+</div>
+<br>
+
+#### .env
+The second file you need to open is the one containing the environment variables, and needs to have the name `.env`. 
+Inside, add the line `BOT_TOKEN=YourDiscordBotTokenHere`. See [here](../creating-bot-account) for help with obtaining the bot token.
+
+The `.env` file will be ignored by commits.
+
+### Run it!
+#### With Docker
+You are now almost ready to run the Python bot. The simplest way to do so is with Docker. 
+
+In your `config.yml` file, set `urls.site` to `"web:8000"`.
+
+<div class="card">
+    <button type="button" class="card-header collapsible">
+        <span class="card-header-title subtitle is-6 my-2 ml-2">Getting started with Docker</span>
+        <span class="card-header-icon">
+            <i class="fas fa-angle-down title is-5" aria-hidden="true"></i>
+        </span>
+    </button>
+    <div class="collapsible-content">
+        <div class="card-content">
+            The requirements for Docker are:
+            <ul>
+                <li><a href="https://docs.docker.com/install">Docker CE</a></li>
+                <li>Docker Compose. If you're using macOS and Windows, this already comes bundled with the previous installation. Otherwise, you can download it either from the <a href="https://docs.docker.com/compose/install">website</a>, or by running <code>pip install docker-compose</code>.</li>
+            </ul>
+            <p>If you get any Docker related errors, reference the <a href="../docker#possible-issues">Possible Issue</a> section of the Docker page.</p>
+        </div>
+    </div>
+</div>
+<br>
+
+Assuming you have Docker installed, enter the cloned repo in the command line and type `docker-compose up`.
```

Comment:
Can we also specify that you need to have Docker running? It's a small thing that seems obvious, but I think it'd be helpful to specify.